### PR TITLE
Note temporary webpack issue

### PR DESCRIPTION
**Pull request by angus-c**
Resolves #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,3 +55,117 @@ app.register(SwPlugin);
 ```
 
 The browser will automatically register the default service worker on page load.
+
+---
+
+### *Temporary Default Service Worker
```
Comment (KevinGrandon) [inline README.md]: Are we sure we want to have this content in the README? Just wondering if a standalone .js file would be better.

**Comment by chrisdothtml** [comment]
Resolves #76 not #77 right? 😀

Also, is there a link you could provide to the webpack issue?

File: README.md (hunk 1 above)
Comment (angus-c) [reply README.md]: the problem is we can't use imports in sw.js right now

Comment (angus-c) [reply README.md]: alternatively we add it to the scaffold, but that won't help externals

**Comment by angus-c** [comment]
Ok I think I'll abandon this one

**Pull request closed without merging**